src/order_gui.cpp

```diff
@@ -1709,7 +1702,7 @@ static constexpr NWidgetPart _nested_orders_widgets[] = {
 				NWidget(WWT_PUSHTXTBTN, COLOUR_GREY, WID_O_REFIT), SetMinimalSize(186, 12), SetFill(1, 0),
 													SetStringTip(STR_ORDER_REFIT, STR_ORDER_REFIT_TOOLTIP), SetResize(1, 0),
 				NWidget(NWID_BUTTON_DROPDOWN, COLOUR_GREY, WID_O_DEPOT_ACTION), SetMinimalSize(124, 12), SetFill(1, 0),
-													SetStringTip(STR_JUST_STRING), SetResize(1, 0),
+													SetResize(1, 0),
```

Comment:
The previous line no longer needs to wrap, could be made a single line.

Only if you want, though. 🙂

Reply:
I noticed, but also noticed that the wrapping is waaaay far off the regular wrapping for all the other lines too, so decided to leave that alone for now :-)